pkgs/by-name/me/med/package.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, qt6, fetchFromGitHub, cmake, pkg-config, jsoncpp, readline }:
+
+stdenv.mkDerivation rec {
+  pname = "med";
+  version = "3.10.1";
+
+  src = fetchFromGitHub {
+    owner = "allencch";
+    repo = "med";
+    rev = version;
+    sha256 = "sha256-m2lVRSNaklB0Xfqgtyc0lNWXfTD8wTWsE06eGv4FOBE=";
+  };
+
+  nativeBuildInputs = [ qt6.wrapQtAppsHook cmake pkg-config ];
+  buildInputs = [ qt6.qtbase qt6.qttools jsoncpp readline ];
```

Comment:
```suggestion
  buildInputs = [ qt6.qtbase qt6.qttools qt6.qtwayland jsoncpp readline ];
```

Reply:
Thanks for the review, I added qtwayland to the buildInputs.